src/modules/replaceContent.js

```diff
@@ -0,0 +1,29 @@
+import { query } from '../utils.js';
+
+/**
+ * Perform the replacement of content after loading a page.
+ *
+ * This method can be replaced or augmented by plugins to allow pausing.
+ *
+ * It takes an object with the page data as return from `getPageData` and has to
+ * return a Promise that resolves once all content has been replaced and the
+ * site is ready to start animating in the new page.
+ *
+ * @param {object} page The page object
+ * @returns Promise
+ */
+const replaceContent = function({ blocks, title }) {
```

Comment:
Could be an arrow function. But I don't care that much tbh :)

Reply:
I think it needs to be a normal function so we can stick it on the swup instance and it keeps its `this` reference. Not import here, anyway.

Reply:
Interesting... just tested it, you are right! With an arrow function, `this` becomes undefined.